### PR TITLE
Adding manual for Sonoff T1 UK 3 Gang V1.1

### PR DESCRIPTION
**Pull request by nnmalex**
## Description:
This manual is based on the existing 4CH version with necessary changes.

I know there's a pending PR for V1 of the same switch but V1.1 is more difficult to get into flashing mode and behaves weirdly after flashing.

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.

### CODE REVIEW

**Comment by nnmalex** [comment]
Sorry images/.DS_Store has slipped in.

File: devices/sonoff_t1_uk_3gang_v1.1.rst [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+
+.. seo::
+    :description: Instructions for putting Sonoff T1 UK 3 Gang devices into flash mode and installing ESPHome on them.
+    :image: sonoff_t1_uk_3g_v1.1.jpg
+
+ESPHome can also be used with Sonoff T1 UK 3 Gang wireless switches. These devices are
+basically just an ESP8266 chip with 3 relays to control power output ant three backlit capacitive touch buttons to control the relays.
+
+.. note::
+    This instruction is made for T1 UK with the touch board v1.1 which is tricker to get into flash mode
```
Comment (OttoWinter) [inline devices/sonoff_t1_uk_3gang_v1.1.rst]:
```suggestion
    This instruction is made for T1 UK with the touch board v1.1 which is trickier to get into flash mode
```

File: devices/sonoff_t1_uk_3gang_v1.1.rst [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+Now go ahead and connect these pins to your UART to USB bridge as seen in below image. Make sure
+that you connect these correctly, especially the ``VCC33`` and ``GND`` parts as you can otherwise
+destroy the chip.
+
+``VCC33`` should be connected to the ``3V3`` (**not** 5V) pin of the UART bridge, ``GND`` to ``GND``
+and the same with ``RX``/``TX``.
+
+There's no need for soldering - for quick job like one time firmware flashing you can just hold pins in respective holes by hand
+providing ``you are not touching any live contacts``, only the wires.
```
Comment (OttoWinter) [inline devices/sonoff_t1_uk_3gang_v1.1.rst]:
```suggestion
provided **you are not touching any live contacts**, only the wires.
```

File: devices/sonoff_t1_uk_3gang_v1.1.rst [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+the device will start up without accepting any firmware flash attempts. 
+To put ESP8266 into flash mode you need to connect ``GPIO0`` to ``GND`` when the device is powering up.
+
+This is a tricky process with T1 and the best way to do it is to use a wire with pins on either side.
+To do this, while the device is UART bridge is not connected to your USB port, flip the PCB over,
+take a wire and connect the second Ground hole on the PCB (red) to the third from the right bottom leg on the chip as depicted below (yellow) -
+that leg is connected to the GPIO0 on ESP and plug the UART to your USB port.
+
+Keep holding  GND and GPIO0 connected for 2-4 seconds. The T1 UK 3 Gang should now be in a flash mode and should not blink with any LED.
+Touchpads may be lit up.
```
Comment (OttoWinter) [inline devices/sonoff_t1_uk_3gang_v1.1.rst]:
```suggestion
The touchpads may light up.
```

File: devices/sonoff_t1_uk_3gang_v1.1.rst [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+
+.. figure:: images/sonoff_t1_uk_3g_back_plate_v1.1.jpeg
+    :align: center
+
+    Connect red and yellow contacts while powering the boar up.
```
Comment (OttoWinter) [inline devices/sonoff_t1_uk_3gang_v1.1.rst]:
```suggestion
    Connect red and yellow contacts while powering the board.
```
or 
```suggestion
    Connect red and yellow contacts while booting the board.
```

File: devices/sonoff_t1_uk_3gang_v1.1.rst [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+.. figure:: images/sonoff_4ch_upload.png
+    :align: center
+
+Hooray 🎉! You've now successfully uploaded the first ESPHome firmware to your Sonoff T1 UK 3 Gang. And in a moment,
+you will be able to use all of ESPHome's great features with your Sonoff T1 UK 3 Gang. Now you can put your T1 back together and fire up.
+
+.. note::
+    While now your T1 will start up and connect to your WiFi network if you power it up from UART it will not behave normally,
+    it may flash random LEDs, turn on anf off touchpads' backlight and not react on touching touchpads. This will all be fixed once you re-assemble your T1
+    and powerit up from the mains power once safe to do so.
```
Comment (OttoWinter) [inline devices/sonoff_t1_uk_3gang_v1.1.rst]:
```suggestion
    and power it up from the mains power once safe to do so.
```

**Review by OttoWinter — COMMENTED** [review]
- The images appear to be defective, see https://deploy-preview-182--esphome.netlify.com/devices/sonoff_t1_uk_3gang_v1.1 - for me they show up with weird artifacts in the picture (for example in the second image in the middle the color suddenly changes). I can attach a screenshot if you're not seeing this in your browser.
- Please rename the files to `.jpg` (not necessary right now, but I'd like to keep a consistent names for automated scripts down the line)

**Comment by nnmalex** [comment]
I can see the artifacts on the images but local copies are fine. Will double check tomorrow. Thank you for correcting the spelling, I’ll pay more attention to it in the future. 
Noted on the jpg

**Comment by nnmalex** [comment]
Just changing image extensions to .jpg and re-committing did the trick.

**Review by OttoWinter — APPROVED** [review]
Sorry for the delay and thanks very much for the guide!

**Pull request merged**